privacyidea/static/components/user/views/user.html

```diff
@@ -4,7 +4,7 @@
         <div class="col-lg-3">
             <div class="pa-sidebar panel">
                 <ul class="nav nav-pills nav-stacked"
-                    ng-show="loggedInUser.role === 'user'">
+                    ng-show="loggedInUser.role == 'user' && checkRight('updateuser') && checkRight('password_reset')">
```

Comment:
updateuser is fine.
But you need to remove password_reset as mentioned in the comment